include/SDL3/SDL_video.h

```diff
@@ -868,14 +868,34 @@ extern DECLSPEC SDL_Window *SDLCALL SDL_CreateWindowWithPosition(const char *tit
 extern DECLSPEC SDL_Window *SDLCALL SDL_CreatePopupWindow(SDL_Window *parent, int offset_x, int offset_y, int w, int h, Uint32 flags);
 
 /**
- * Create an SDL window from an existing native window.
+ * Create an SDL window from properties representing an existing native window.
  *
- * In some cases (e.g. OpenGL) and on some platforms (e.g. Microsoft Windows)
- * the hint `SDL_HINT_VIDEO_WINDOW_SHARE_PIXEL_FORMAT` needs to be configured
- * before using SDL_CreateWindowFrom().
+ * These are the supported properties:
  *
- * \param data a pointer to driver-dependent window creation data, typically
- *             your native window cast to a void*
+ * On macOS:
```

Comment:
In `testnative.c`, you're setting only the `"data"` property.
Doesn't this contradict `SDL_CreateWindowFrom`'s documentation?

Reply:
Yes, but it's the most convenient in the case of a factory function that returns a void*. It's not going to be the normal application use case, and we need to support that for sdl2-compat, so it's fine here.

Reply:
I see, so it's an undocumented use case. Thanks.